subsys/bluetooth/host/gatt.c

```diff
@@ -876,7 +876,9 @@ static void sc_indicate(uint16_t start, uint16_t end);
 
 static void db_hash_process(struct k_work *work)
 {
-	if (!atomic_test_bit(gatt_sc.flags, DB_HASH_VALID)) {
+	bool new_hash = !atomic_test_bit(gatt_sc.flags, DB_HASH_VALID);
```

Comment:
@jori-nordic, could you take a look at this one?

Reply:
I think I've seen in my test that the run can be scheduled even if `DB_HASH_VALID` is set.

Reply:
Discussed offline.
I think it was before hash was initially calculated after finished settings load. Still we can leave it as it is for now, because it should not break anything.